spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+RSpec.describe IssuesController, :type => :controller do
```

Comment:
Same comment as for the "rails-controller-testing" gem, I wonder if we could use Requests specs instead?